rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java

```diff
@@ -59,9 +59,8 @@ protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder request) th
         int responseCode;
         try (ByteBufOutputStream os = new ByteBufOutputStream(responseContent);
              ByteBufInputStream is = new ByteBufInputStream(request.content().retain())){
-
             responseCode = jsonRpcServer.handleRequest(is, os);
-        } catch (Exception e) {
+        } catch (Throwable e) {
```

Comment:
Any reason to this change?

Reply:
To catch stackoverflow... but i was thinking that maybe has no sense.